charts/internal/calico/templates/typha/deployment-calico-typha.yaml

```diff
@@ -49,6 +49,9 @@ spec:
     spec:
       nodeSelector:
         beta.kubernetes.io/os: linux
```

Comment:
I see you have removed this selector for the `calico-kube-controller`, is it possible to remove it also for `calico-typha`?

Reply:
I've removed it by accident. I'll readd it

Reply:
Removed the selector